e2e_test/backwards-compat-tests/scripts/utils.sh

```diff
@@ -244,6 +244,15 @@ seed_old_cluster() {
   echo "--- KAFKA TEST: Validating old cluster"
   sqllogictest -d dev -h localhost -p 4566 "$TEST_DIR/kafka/validate_original.slt"
 
+  # Test version columns backwards compatibility, if OLD_VERSION <= 2.6.0
+  if version_le "$OLD_VERSION" "2.6.0"; then
+    echo "--- VERSION COLUMNS TEST: Seeding old cluster with data"
+    sqllogictest -d dev -h localhost -p 4566 "$TEST_DIR/version-columns/seed.slt"
+
+    echo "--- VERSION COLUMNS TEST: Validating old cluster"
+    sqllogictest -d dev -h localhost -p 4566 "$TEST_DIR/version-columns/validate_original.slt"
+  fi
+
```

Comment:
I also added some backwards compact test. PTAL if I'm doing this right @yezizp2012